### DISCUSSION TITLE
关于IC2中尝试使用<批量工作台>而导致游戏崩溃bug

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#31] [【漏洞反馈】Bugs Report]
[错误报告-2023-5-7_16.26.42.zip](https://github.com/AbbottMc/IndustrialCraft2/files/11414245/-2023-5-7_16.26.42.zip)

![image](https://user-images.githubusercontent.com/132697942/236666694-7ad9a8f0-689a-4428-ac1c-2c918cd091c0.png)

使用模组版本：1.industrialcraft-2-2.9.84+ex119-1.19.2-forge
2.[1.19.2]Jei物品管理器(forge11.3.0.262)

游戏版本：1.19.2 forge-43.2.0 Op-I1-pre4

问题描述：尝试使用<批量工作台>后游戏崩溃。该机器可以正常交互并输入<合成路径>，但当我尝试将<合成原料>放入其中时，游戏崩溃。重新进入游戏后，发现<合成路径>及其<合成路径的最终产物>均消失，尝试放入其中的<合成原料>并未返还背包，直接消失：
![image](https://user-images.githubusercontent.com/132697942/236667051-dcb6521c-e7ff-48ac-87e0-4b3f9edf6020.png)

**Comment by AbbottMc** [comment]
已修复
